src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx

```diff
@@ -42,7 +43,10 @@ export const RemoveLiquidityModal = () => {
   const styles = useRemoveLiquidityModalStyles();
   const assetsList = useAssetsListSelector();
 
-  const lpToken = assetsList.find(token => getTokenSlug(token) === LIQUIDITY_BAKING_LP_SLUG) ?? emptyToken;
+  const lpToken = assetsList.find(token => getTokenSlug(token) === LIQUIDITY_BAKING_LP_SLUG) ?? {
+    ...emptyToken,
+    ...LIQUIDITY_BAKING_LP_METADATA
+  };
```

Comment:
we would not be able to reuse this on QS.
call `loadTokenMetadataActions.submit` if there is no token in `assetsList`